meta/main.yml

```diff
@@ -19,5 +19,8 @@ galaxy_info:
   galaxy_tags:
     - mattermost
     - github
+    - push
+    - webhook
+    - push notification
```

Comment:
Have you verified that spaces are allowed in tags?

Reply:
they are not fixed